homeassistant/components/zcs_azzurro_inverter/__init__.py

```diff
@@ -0,0 +1,49 @@
+"""The zcs_azzurro integration."""
+from __future__ import annotations
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import DOMAIN
+
+PLATFORMS: list[Platform] = [Platform.SENSOR]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up zcs_azzurro from a config entry."""
+
+    hass.data.setdefault(DOMAIN, {})
+
+    await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
+
+    return True
```

Comment:
you need to set hass data [entry] like you do in your async_setup_entry in sensor.

But you need to do it here, not in sensor.